deploy/rbac-permissions-operator/00-dedicated-admins-project.ClusterRole.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dedicated-admins-project
```

Comment:
I updated this Role yesterday in dedicated-admin adding permission for strimzi kafka CRs, could you please incorporate it here?

Reply:
added at bottom of file